contents/blog/analytics-pricing.md

```diff
@@ -0,0 +1,130 @@
+
+| **Monthly events** | **Identified events** | **Anonymous events** | **Change** |
+|--------------------|--------------------------|-----------------------------------|----------------|
+| <strong class="text-15px">0-1 million</strong>      | Free | Free  | &nbsp |
+| <strong class="text-15px">1-2 million</strong>      | <span class="text-[15px] font-semibold">$0.0002480</span><span class="text-sm opacity-70">/event</span> | <span class="text-[15px] font-semibold">$0.0000500</span><span class="text-sm opacity-70">/event</span>  | <strong class="text-green">80% cheaper</strong> |
+| <strong class="text-15px">2-15 million</strong>     | <span class="text-[15px] font-semibold">$0.0001040</span><span class="text-sm opacity-70">/event</span> | <span class="text-[15px] font-semibold">$0.0000343</span><span class="text-sm opacity-70">/event</span>  | <strong class="text-green">67% cheaper</strong> |
+| <strong class="text-15px">15-50 million</strong>    | <span class="text-[15px] font-semibold">$0.0000655</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000295</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">55% cheaper</strong> |
+| <strong class="text-15px">50-100 million</strong>   | <span class="text-[15px] font-semibold">$0.0000364</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000218</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">40% cheaper</strong> |
+| <strong class="text-15px">100-250 million</strong>  | <span class="text-[15px] font-semibold">$0.0000187</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000150</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">20% cheaper</strong> |
+| <strong class="text-15px">250+ million</strong>     | <span class="text-[15px] font-semibold">$0.0000100</span><span class="text-sm opacity-70">/event</span>  | <span class="text-[15px] font-semibold">$0.0000090</span><span class="text-sm opacity-70">/event</span> | <strong class="text-green">10% cheaper</strong> |
+
+## When should I use anonymous events?
+Anonymous events are ideal for tracking users who aren't logged in, or for use cases where you only need to analyze behavior at an aggregate level, such as:
+
+- Visitors to your marketing website
+- Readers of a news website
+- Shoppers on an e-commerce store
+- Users of a B2C mobile app
+- Tracking API and backend events
+- Events generated by LLMs
+- Error monitoring
+
+If you mostly track logged-in users of a B2B product, however, you should continue to do so. 
+
+We're supporting both because we want people to have the level of analysis they need for their use case at a fair price, rather than forcing them to pay more when they only need basic data on individual users.
```

Comment:
This sentence is long and confusing and we explain it better elsewhere (earlier)
```suggestion
```